Jenkinsfile.parameters

```diff
@@ -0,0 +1,111 @@
+env.LANG="en_US.UTF-8"
+env.LANGUAGE="en_US.UTF-8"
+env.LC_ALL="en_US.UTF-8"
+node ('macos1') {
+  def apkUrl = ''
+  def ipaUrl = ''
+  def testPassed = true
+  def branch;
+
+   load "$HOME/env.groovy"
+
+  try {
+
+    stage('Git & Dependencies') {
+      slackSend color: 'good', message: BRANCH_NAME + ' build started. ' + env.BUILD_URL
+      git([url: 'https://github.com/' + REPO + '/status-react.git', branch: BRANCH_NAME])
+      // Checkout master because used for iOS Plist version information
+      sh 'git checkout -- .'
+      sh 'git checkout master'
+      sh 'git checkout ' + BRANCH_NAME
+      sh 'rm -rf node_modules'
+
+      // NOTE(oskarth): This is just a copy of the real Jenkins config,
+      // so don't expect changes to show up in Jenkins by editing this file.
+      // We should make sure they are in sync though.
+      //
+      // Assume all parameters are set in Jenkins 'Parameterized build'
+      // TODO(oskarth): Consider read/write from .env to avoid having to specify in Jenkins again
+      // sh 'cp .env.jenkins .env'
+      sh 'echo TESTFAIRY_ENABLED='         + TESTFAIRY_ENABLED         + '>>' + '.env'
+      sh 'echo NOTIFICATIONS_WIP_ENABLED=' + NOTIFICATIONS_WIP_ENABLED + '>>' + '.env'
+      sh 'echo DEBUG_LOGS_ENABLED='        + DEBUG_LOGS_ENABLED        + '>>' + '.env'
+      sh 'echo STUB_STATUS_GO='            + STUB_STATUS_GO            + '>>' + '.env'
+      sh 'echo ETHEREUM_DEV_CLUSTER='      + ETHEREUM_DEV_CLUSTER      + '>>' + '.env'
+      sh 'echo MAINNET_NETWORKS_ENABLED='  + MAINNET_NETWORKS_ENABLED  + '>>' + '.env'
+      sh 'echo ERC20_ENABLED='             + ERC20_ENABLED             + '>>' + '.env'
+      sh 'echo LOG_LEVEL='                 + LOG_LEVEL                 + '>>' + '.env'
+      sh 'echo "**********************************************************************"'
+      sh 'echo PARAMETERIZED BUILD - USING CUSTOM ENVIRONMENT'
+      sh 'cat .env'
+      sh 'echo "**********************************************************************"'
+
+      sh 'lein deps && npm install && ./re-natal deps'
+      sh 'sed -i "" "s/301000/1201000/g" node_modules/react-native/packager/src/JSTransformer/index.js'
```

Comment:
I think we should remove this line now, also once react-native will be updated this won't work anymore as the packager is now a separate project

Reply:
@yenda Can you raise an issue and we test/do this in a separate PR and for all Jenkinsfiles? This is not in scope of this PR IMO (and also need to update it in Jenkins).

Reply:
let's just try it on jenkins first

Reply:
It was already commented out in jenkins until @oskarth updated it with that script